tools/tests/populate_and_build.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env python3
+
+""" Test `antler-proj add` and `antler proj update` commands. """
+
+import os
+import shutil
+import subprocess
+
+from util import APROJ_EXE
+
+PROJECT_NAME="pandb_contract"
+PROJECT_PATH=os.path.join("./",PROJECT_NAME)
+SOURCE_PATH=os.path.join("@CMAKE_CURRENT_SOURCE_DIR@/tests",PROJECT_NAME)
+
+
+def test_populate_and_build():
+    """ Setup a clean project, then populate and build it.
+    """
+    shutil.rmtree(PROJECT_PATH, ignore_errors=True)
+    assert shutil.copytree(SOURCE_PATH, PROJECT_PATH)
+
+    subprocess.run([APROJ_EXE,"populate", PROJECT_PATH], check=True)
+    subprocess.run([APROJ_EXE,"build", PROJECT_PATH], check=True)
```

Comment:
I also suggest to print full output in case of failure, otherwise that is long debug journey for developer
```suggestion
    result = subprocess.run([APROJ_EXE,"build", PROJECT_PATH], capture_output=True, text=True)
    print(result.stdout)
    print(result.stderr)

    sys.exit(result.returncode)
```

Reply:
I like it, I'll add that in tomorrow!

Reply:
Fixed in 1a0a18b.